src/components/Checkbox/Checkbox.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <label class="checkbox" 
+    :class="[{'checkbox__input--disabled': disabled},
+            {'checkbox__input--same-line': sameLine}]"
+  >
+    <input
+      :class="['checkbox__input', {'checkbox__input--error': error}]"
+      type="checkbox"
+      v-model="computedVModel"
+      :value="nativeValue"
```

Comment:
I'm still really struggling to understand `nativeValue` vs. `value` and would love to chat about this during pairing to see if we can simplify this at all.